integration_tests/src/main/python/fastparquet_compatibility_test.py

```diff
@@ -107,8 +107,12 @@ def read_with_fastparquet_or_plugin(spark):
     pytest.param(IntegerGen(nullable=True),
                  marks=pytest.mark.xfail(reason="Nullables cause merge errors, when converting to Spark dataframe")),
     LongGen(nullable=False),
-    FloatGen(nullable=False),
-    DoubleGen(nullable=False),
+    pytest.param(FloatGen(nullable=False),
+                 marks=pytest.mark.xfail(is_databricks_runtime(),
+                                         reason="https://github.com/NVIDIA/spark-rapids/issues/9778")),
```

Comment:
I was thinking of including the following:
```suggestion
    pytest.param(FloatGen(nullable=False),
                 marks=pytest.mark.xfail(is_databricks_runtime(),
                                         reason="https://github.com/NVIDIA/spark-rapids/issues/9778")),
    FloatGen(nullable=False, no_nans=True),
```
Not strictly in the purview of this change. I can add this as a follow-on.